app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -253,6 +253,15 @@ class Settings private constructor(
         default = true
     )
 
+    val shouldShowSearchSuggestionsInPrivate by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_show_search_suggestions_in_private),
+        default = false
+    )
+
+    val showSearchSuggestionsInPrivateOnboardingFinished by booleanPreference(
+            appContext.getPreferenceKey(R.string.pref_key_show_search_suggestions_in_private_onboarding),
+            false)
```

Comment:
nit: Use named arguments for the boolean
```suggestion
        default = false
    )
```